lib/generators/dockerfile_generator.rb

```diff
@@ -737,10 +738,10 @@ def base_requirements
 
   def build_packages
     # start with the essentials
-    packages = %w(build-essential)
+    packages = %w(build-essential libyaml-dev)
     packages += @@packages["build"] if @@packages["build"]
     packages += %w(nodejs npm) if (node_version == "lts") && (not using_execjs?)
-    packages << "libyaml-dev" if options.fullstaq?
+    packages << "libffi-dev" if  @@packages["ffi"]
```

Comment:
I think this should have been `@gemfile.include?("ffi")`

Reply:
Good catch.  Fixed in https://github.com/fly-apps/dockerfile-rails/pull/132